Makefile

```diff
@@ -462,21 +463,22 @@ test-and-cover:
 		tail -n +2 profile_tmp.cov >> profile.cov; \
 	done
 
-android-lib: docker-golang-android genconfig
+android-lib-dev: docker-mobile
```

Comment:
So, I still would have two separate tasks, but DRY by one depending on the other. How does this sound?